tests/gnrc_dhcpv6_client_auto_init/README.md

```diff
@@ -0,0 +1,31 @@
+# `gnrc_dhcpv6_client_auto_init` test
+
+This test utilizes [scapy] to test the DHCPv6 client configuration for a
+device which uses auto initialization for requesting a non-temporary address
+from a DHCPv6 server (`IA_NA`).
```

Comment:
The name seems a bit weird.. prefix delegation and MUD also use auto initialization. How about calling the test `gnrc_dhcpv6_client_ia_na`?

Reply:
I have renamed both folder and the heading of the readme file in aaa750b3b41fb8244155af6536b596ff25112be2